ingest-app/src/cmr/ingest/data/bulk_update.clj

```diff
@@ -36,23 +37,31 @@
 
   (get-provider-bulk-update-status
     [db provider-id]
-    ;; Returns a list of bulk update statuses for the provider
-    (let [statuses (su/query db (su/build (su/select [:task-id :status :status-message :request-json-body]
-                                            (su/from "bulk_update_task_status")
-                                            (su/where `(= :provider-id ~provider-id)))))
-          statuses (map util/map-keys->kebab-case statuses)]
-     (map #(update % :request-json-body util/gzip-blob->string)
-          statuses)))
+    (j/with-db-transaction
+     [conn db]
+     ;; Returns a list of bulk update statuses for the provider
+     (let [stmt (su/build (su/select [:created-at :task-id :status :status-message :request-json-body]
+                            (su/from "bulk_update_task_status")
```

Comment:
Minor: we should make the name of the table a constant defined in this file. The indentation looks off as well - this line should be directly below the start of the argument above.

Reply:
Quick look at the Oracle schemas in a running local instance:
 * `CMR_INGEST.BULK_UPDATE_COLL_STATUS` doesn't have a `CREATED_AT` column
 * `CMR_INGEST.BULK_UPDATE_TASK_STATUS` does ...

Reply:
Correct. we only want to use the created_at from bulk_update_task_status table. Done.